Jenkinsfile

```diff
@@ -99,6 +105,40 @@ try {
         containers = limit_builds(containers)
         // launch jenkins agents to support the container scale!
         spotScaleSwarm layer_name: 'swarm-ide', instance_count: containers.size(), duration_seconds: 7000
+
+        // create the version we're about to build
+        node('ide') {
```

Comment:
best to set this to `node('docker')` since it's really lightweight and wont have to wait until the ide nodes have arrived in order to execute!